README.md

```diff
@@ -118,6 +118,7 @@ healthbc - settings.healthbc.sh
   - ../openshift/templates/issuer-web/issuer-web-deploy.json
   - ../openshift/templates/wallet/wallet-deploy.json
 ```
+We have moved to creating a single wallet pod across all essential services for greater resource sharing. The shared wallet is deployed from the default profile and used by all agents.
```

Comment:
Nit.: Maybe add something like `please ensure the wallet component is deployed BEFORE attempting to deploy any issuer agent` to make it inequivocably clear.

Reply:
change completed